src/index.tsx

```diff
@@ -57,27 +57,27 @@ const App: React.FC = () => {
     })
     setClient(client)
 
-    // persistCache({
-    //   cache,
-    //   storage: window.localStorage,
-    //   /**
-    //    * Storage options.
-    //    */
-    //   // Maximum size of cache to persist (in bytes).
-    //   // Defaults to 1048576 (1 MB). For unlimited cache size, provide false.
-    //   // If exceeded, persistence will pause and app will start up cold on next launch.
-    //   maxSize: false,
+    persistCache({
+      cache,
+      storage: window.localStorage,
+      /**
+       * Storage options.
+       */
+      // Maximum size of cache to persist (in bytes).
+      // Defaults to 1048576 (1 MB). For unlimited cache size, provide false.
+      // If exceeded, persistence will pause and app will start up cold on next launch.
+      // maxSize: false,
```

Comment:
Not really sure if we should set a maximum or not.

Reply:
I went on demo4, and clicked in quite a few places, probably spent 1 minutes clicked through paginations and different filters, got to 200kb, i think it would be ok to set to like 3mb

Reply:
Cool: f0f5da98